vector/src/main/res/values/config.xml

```diff
@@ -7,6 +7,12 @@
     <string name="matrix_org_server_url" translatable="false">https://matrix.org</string>
     <string name="piwik_server_url" translatable="false">https://piwik.riot.im</string>
     <string name="bug_report_url" translatable="false">https://riot.im/bugreports/submit</string>
+    <string name="fallback_stun_server_url" translatable="false">turn.matrix.org</string>
+    <string-array name="fallback_ice_servers" translatable="false">
+        <item>turn:turn.matrix.org:3478?transport=udp</item>
```

Comment:
I don't think this will work, turn.matrix.org requires auth.
Just use stun: would work

Reply:
`fallback_ice_servers` can just be removed now I think

Reply:
yep, I've removed it